app/browser/api/ledger.js

```diff
@@ -609,15 +609,33 @@ const roundToTarget = (l, target, property) => {
     })
 }
 
+// Removes entries older than 30 days and entries that have 0 visits
+const pruneSynopsis = (state) => {
+  if (!synopsis) {
+    return state
+  }
+
+  const json = synopsis.toJSON()
+  if (!json || !json.publishers) {
+    return state
+  }
+
+  return ledgerState.saveSynopsis(state, json.publishers)
+}
+
 // TODO we should convert this function and all related ones into immutable
 // TODO merge publishers and publisherData that is created in getPublisherData
 // so that we don't need to create new Map every single time
-const synopsisNormalizer = (state, changedPublisher, returnState) => {
+const synopsisNormalizer = (state, changedPublisher, returnState = true, prune = false) => {
   let dataPinned = [] // change to list
   let dataUnPinned = [] // change to list
   let dataExcluded = [] // change to list
   const scorekeeper = ledgerState.getSynopsisOption(state, 'scorekeeper')
 
+  if (!prune) {
```

Comment:
is this condition correct? I would think you would only call prune if prune is true?  Is the variable meaning IsPruned (ex: is it already pruned?).  In that sense, this check would make sense.  It's a little confusing

Reply:
no it's not